Xamarin.Forms.Core/Items/CarouselView.cs

```diff
@@ -259,5 +262,11 @@ public void SetIsDragging(bool value)
 		{
 			SetValue(IsDraggingPropertyKey, value);
 		}
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void SetIsScrolling(bool value)
```

Comment:
IsScrolling is not a BBPKey, so there is no need to expose a Setter

Reply:
Your are right, removed it. Thanks!